src/unix/linux_like/mod.rs

```diff
@@ -1254,6 +1254,10 @@ f! {
         (status & 0x80) != 0
     }
 
+    pub fn W_EXITCODE(ret: ::c_int, sig: ::c_int) -> ::c_int {
+        (ret << 8) | sig
+    }
+
```

Comment:
While doing this, maybe you could also add `W_STOPCODE`?
```suggestion

    pub fn W_STOPCODE(sig: ::c_int) -> ::c_int {
        (sig << 8) | 0x7f
    }

```

Reply:
Done.